Disable autoupdate and update New version manifest from local repository

shanmugavel-s-2352 opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/189387)

shanmugavel-s-2352 committed:
Update Zoho.TrueSync.yaml

shanmugavel-s-2352 committed:
Disable autoupdate for Zoho.TrueSync

Shan3747 commented:
Hi Reviewer, 
  I want to stop autosyncing of new version via wingetbot as we get lot of requests for download which are invalid. We want to update each version manually and hence requesting to stop autosyncing via bot. I have updated new version manifests and attached here. Do let me know if anything needed to update each version under each directory. Thanks for assistance. 
![Screenshot 2024-11-07 144856](https://github.com/user-attachments/assets/f9c96f36-87da-41eb-9041-e6c34072c244)

shanmugavel-s-2352 committed:
Updating newer version manually

Want to disable Autoupdate of Application via WingetBot
and update each version via new manifest.

shanmugavel-s-2352 committed:
Update Zoho.TrueSync.installer.yaml

Updated hash value

stephengillie commented:
Close with reason: Duplicate of #189331;
